src/bitmessagekivy/mpybit.py

```diff
@@ -206,6 +206,56 @@ def load_screen_callback(self, instance, dt=0):
             except Exception as e:
                 self.root.ids.id_trash.children[0].children[1].active = False
 
+    @staticmethod
+    def current_address_label(current_add_label=None, current_addr=None):
```

Comment:
split into 3 functions
1. get the current address
2. get data for specified address
3. format label (deterministic, use `@functools.lru_cache` decorator)